src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/DestinationInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Export.Models
+{
+    public class DestinationInfo
+    {
+        public DestinationInfo(string destinationType, string destinationConnectionString)
+        {
+            EnsureArg.IsNotNullOrWhiteSpace(destinationType, nameof(destinationType));
+            EnsureArg.IsNotNullOrWhiteSpace(destinationConnectionString, nameof(destinationConnectionString));
+
+            DestinationType = destinationType;
+            DestinationConnectionString = destinationConnectionString;
+        }
+
+        [JsonConstructor]
+        protected DestinationInfo()
+        {
+        }
+
+        public string DestinationType { get; private set; }
```

Comment:
I think this will serialize the properties with upper case. Is that intentional?

Reply:
Added JsonPropery with explicit camelCase names